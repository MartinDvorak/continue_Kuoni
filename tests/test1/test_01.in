nejaky test na STDIN pro python
asd
	