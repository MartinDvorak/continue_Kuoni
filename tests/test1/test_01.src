.IPPcode18

CREATEFRAME 
DEFVAR LF@aaa