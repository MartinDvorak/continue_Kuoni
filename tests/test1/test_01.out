<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode18">
  <instruction order="1" opcode="CREATEFRAME"/>
  <instruction order="2" opcode="DEFVAR">
    <arg1 type="var">LF@aaa</arg1>
  </instruction>
</program>
